tor/strings/posts.py

```diff
@@ -14,15 +14,18 @@
     'wiki/checklist) that might help you out -- not just for newbies!'
     '\n\nThis is a(n) {post_type} post, so please use the following formatting:'
     '\n\n---\n\n{formatting}\n\n---'
-    '\n\nFooter\n---\n\nWhen you\'re done, please put the following footer at '
-    'the **bottom** of your post:\n\n---\n\n{header}\n\n---'
+    '\n\nReporting\n---'  # this is a header
```

Comment:
wouldn't it be better to use `## Reporting` style headers in this case? The underline style seems to only be for visual purposes, which doesn't really work in this case.

Or we could wrap the whole thing in a `"""multi line comment"""` which would also solve having to insert all the apostrophes and `\n`s. Idk it's only a matter of code style

Reply:
Yes, that is a change we should eventually make. I think it would be good to go through and change all our messages to multiline comments eventually, but that should be its own thing.